pkgs/development/compilers/flutter/engine/package.nix

```
@@ -0,0 +1,360 @@
+{ lib,
+  callPackage,
+  writeText,
+  symlinkJoin,
+  cacert,
+  targetPlatform,
+  hostPlatform,
+  darwin,
+  clang,
+  llvm,
+  tools ? callPackage ./tools.nix { inherit hostPlatform; },
+  stdenv,
+  stdenvNoCC,
+  runCommand,
+  fetchurl,
+  patchelf,
+  xorg,
+  libglvnd,
+  libepoxy,
+  wayland,
+  freetype,
+  pango,
+  glib,
+  harfbuzz,
+  cairo,
+  gdk-pixbuf,
+  at-spi2-atk,
+  zlib,
+  gtk3,
+  curl,
+  pkg-config,
+  ninja,
+  python3,
+  git,
+  version,
+  dartSdkVersion,
+  hashes,
+  patches,
+  url,
+  runtimeMode ? "release",
+  isUnoptimized ? false,
```

```suggestion
  isOptimized ? true,
```
double negation is hard to understand usually

---

Done